src/java.desktop/share/classes/sun/swing/SwingUtilities2.java

```diff
@@ -334,15 +334,15 @@ public static FontMetrics getFontMetrics(JComponent c, Graphics g) {
      * painting.  If a Graphics is not available the JComponent method of
      * the same name should be used.
      * <p>
-     * Callers should pass in a non-null JComonent, the exception
+     * Callers should pass in a non-null JComponent, the exception
      * to this is if a JComponent is not readily available at the time of
      * painting.
      * <p>
      * This does not necessarily return the FontMetrics from the
      * Graphics.
      *
      * @param c JComponent requesting FontMetrics, may be null
-     * @param c Graphics Graphics
+     * @param g Graphics Graphics
      * @param font Font to get FontMetrics for
      */
     @SuppressWarnings("deprecation")
```

Comment:
Just below:
```diff
-            // from the widget to layout out text, otherwise we can get
+            // from the widget to lay out text, otherwise we can get
```